mono/mini/interp/interp.c

```diff
@@ -3115,8 +3124,7 @@ mono_interp_newobj (
 				g_assert (exc);
 				return exc;
 			}
-			frame_objref (frame) = mono_object_new_checked (imethod->domain, newobj_class, error);
-			mono_error_cleanup (error); /* FIXME: don't swallow the error */
+			frame_objref (frame) = mono_interp_new (imethod->domain, newobj_class); // FIXME: do not swallow the error
```

Comment:
nit due to merge: Previously this was inline and need to call to get the error allocated.
Now it is not inlined, so we could inline the "regular" call with a local MonoError.
i.e. save one function call. Probably.